pkg/controller/operator/operator_controller.go

```diff
@@ -81,6 +81,7 @@ func (r *ReconcileOperator) Reconcile(request reconcile.Request) (reconcile.Resu
 	operator := &kudov1alpha1.Operator{}
 	err := r.Get(context.TODO(), request.NamespacedName, operator)
 	if err != nil {
+		log.Printf("Operator Controller: Error getting operator %v", request.NamespacedName)
```

Comment:
Do we want to log this even if the error is `NotFound`? If we don't, then we could move this down a few lines.

```suggestion
		log.Printf("OperatorController: Error getting operator %v", request.NamespacedName)
```